packages/button/docs/StyledButton.module.scss

```diff
@@ -0,0 +1,7 @@
+// Does not override combined styles (eg. .default + .isReversed)
+// and not StyledIconButton styles
```

Comment:
Unlikely to actually be a problem though - how often does one need to override the colours and padding? Worst case scenario, they could use `!important`